scripts/ci/after_success.sh

```diff
@@ -16,6 +16,16 @@ fi
 cd sentinel && npm install && cd .. && \
 cd api && npm install && cd ..
 
+function tagSubmodules {
+    cd api
+    git tag $TRAVIS_TAG
+    git push --tags
+    cd ../sentinel
+    git tag $TRAVIS_TAG
```

Comment:
I mean. We probably can.

Sigh. Fine. I'll learn how stupid @#$@% bash scripting works.

;-)